database-emulator/javascript-quickstart/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "database-emulator-javascript-quickstart",
+  "version": "1.0.1",
+  "description": "Real Time Database emulator testing",
+  "scripts": {
+    "test": "mocha"
+  },
+  "author": "ryanpbrewster@google.com",
+  "license": "MIT",
+  "devDependencies": {
+    "@firebase/app": "^0.3.4",
+    "@firebase/database": "^0.3.6",
+    "@firebase/testing": "file:firebase-testing-v0.2.0.tgz",
+    "mocha": "^4.1.0"
```

Comment:
For this quickstart we may want to set these deps to specific versions rather than ranges

Reply:
I believe there were "*" versions in the firestore package.json which I will resolve to real versions as well.